javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,34 @@
+const class07Students = [];
+function addStudentToClass(studentName = String) {
```

Comment:
This `= String` does not make a lot of sense here. It basically sets the default value to being the string "type". I'd recommend removing it. 🙂

(This explanation is overly simplified! If you want to read more about this, look up classes and constructors. To use an example you are familiar with: `Date` is a class and passing `Date` as a variable is like passing its constructor, i.e., the code called when you run `new Date()`, around. Don't worry if this is meaningless to you now; we will get to it later! 😄)

Reply:
got it!
a little complicated 😄 
thanks.